test/validators/movieget.test.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const Joi               = require('joi');
+const MovieGetValidator = require('../../lib/validators/movieget');
```

Comment:
Could we separate these two imports with a newline to distinguish between third party packages and internal libraries?